packages/kit/types/ambient-modules.d.ts

```diff
@@ -1,5 +1,12 @@
 /* eslint-disable import/no-duplicates */
 
+declare namespace App {
+	interface Locals {}
+	interface Platform {}
+	interface Session {}
+	interface Stuff {}
```

Comment:
Any possibility that someone would want this to be a primitive type which can't be expressed as an object-like interface?

Reply:
No, they're all required to be objects